airflow/providers/amazon/aws/executors/batch/batch_executor.py

```diff
@@ -292,12 +292,13 @@ def attempt_submit_jobs(self):
 
             if failure_reason:
                 if attempt_number >= int(self.__class__.MAX_SUBMIT_JOB_ATTEMPTS):
-                    self.send_message_to_task_logs(
-                        logging.ERROR,
-                        "This job has been unsuccessfully attempted too many times (%s). Dropping the task. Reason: %s",
-                        attempt_number,
-                        failure_reason,
-                        ti=key,
+                    self.log_task_event(
+                        record=Log(
+                            event="executor queue failure",
```

Comment:
```suggestion
                            event="executor job failure",
```

Reply:
it's a failure to queue though right?  or perhaps submit is better?

Reply:
It is a failure to submit/execute/run a Batch job that has been dequeued. But the failure is not coming from the queue, it is just whatever job we got from the queue failed to submit/execute/run

Reply:
should this have the same `ecs task submit failure` as the other one? or should these events have different names?

Reply:
I think `ecs task submit failure` makes sense as well here

Reply:
oh but it is aws batch here. `batch submit failure`?

Reply:
Good catch! `batch job submit failure` maybe?